File: bab_11_class/Student.py
#Simpan dengan nama file Student.py
class Student:
    def __init__(self):
        self.id = 0
        self.nama = ""
        self.grade = '\0'

    # Method untuk menambahkan data student
    def add_data(self, new_id, new_name, new_grade):
        self.id = new_id
        self.nama = new_name
        self.grade = new_grade

    # Method untuk mengedit data student
    def edit_data(self, new_name, new_grade):
        self.nama = new_name
        self.grade = new_grade

    # Method untuk menghapus data student
    def remove_data(self):
        self.id = 0
        self.nama = ""
        self.grade = '\0'

File: bab_11_class/Student.cpp
#include "Student.h"
#include <iostream>

// Fungsi untuk menambahkan data student
void Student::addData(int newId, const std::string& newName, char newGrade) {
    id = newId;
    nama = newName;
    grade = newGrade;
}

// Fungsi untuk mengedit data student
void Student::editData(const std::string& newName, char newGrade) {
    nama = newName;
    grade = newGrade;
}

// Fungsi untuk menghapus data student
void Student::removeData() {
    id = 0;
    nama = "";
    grade = '\0';
}
